common/formatDate.js

```diff
@@ -0,0 +1,19 @@
+function formatDate(date) {
+  const day = addZero(date.getDay());
+  const month = addZero(date.getMonth() + 1);
+  const year = addZero(date.getFullYear());
+
+  const hours = addZero(date.getHours());
+  const minutes = addZero(date.getMinutes());
+  const seconds = addZero(date.getSeconds());
+
+  return `${day}-${month}-${year} ${hours}:${minutes}:${seconds}`;
+}
+
+function addZero(value) {
+  return value < 10 ? `0${value}` : value;
+}
+
+export function getCurrentDate() {

```

Comment:
Review this library: [https://www.npmjs.com/package/moment](https://www.npmjs.com/package/moment)
it is very helpful when you are dealing with dates

Reply:
done, added this library